src/libs/ajax.js

```diff
@@ -889,6 +889,11 @@ const Submissions = signal => ({
   queueStatus: async () => {
     const res = await fetchRawls('submissions/queueStatus', _.merge(authOpts(), { signal }))
     return res.json()
+  },
+
+  cromwellVersion: async () => {
+    const res = await fetchOk(`${getConfig().rawlsUrlRoot}/version/executionEngine`, { signal })
```

Comment:
TOL: It's too bad we have to use `fetchOk` here, since `fetchRawls` assumes a prefix. Nothing to be done about it without a refactor, though.

Reply:
Yeah...